src/components/widgets/WaterTemperatureWidget.tsx

```diff
@@ -0,0 +1,38 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize, texts } from '../../config';
+import { useHomeRefresh } from '../../hooks';
+import { useWaterTemperature } from '../../hooks/waterTemperature';
+import { QUERY_TYPES } from '../../queries';
+import { ScreenName, WidgetProps } from '../../types';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WaterTemperatureWidget = ({ text, additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+  const { temperature, refresh } = useWaterTemperature();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Html, {
+      title: text ?? texts.waterTemperature.headerTitle,
```

Comment:
so the title will be always the `text` right, because this is always given. if there would be no `text`, there would be no widget title on the home screen. in this case, we will never have the longer page title, right? maybe we want to add a new additional prop for the `staticContentTitle`?

Reply:
Yeah, that'll be better.